projects/valtimo/plugin/src/lib/plugins/open-zaak/open-zaak-plugin.specification.ts

```diff
@@ -28,24 +28,39 @@ const openZaakPluginSpecification: PluginSpecification = {
       description:
         'OpenZaak is een productiewaardig API platform die de API-standaard voor zaakgericht werken implementeert (de ZGW-API’s).',
       configurationTitle: 'Configuratienaam',
+      configurationTitleTooltip:
+        'Onder deze naam zal de plugin te herkennen zijn in de rest van de applicatie',
       clientId: 'Client ID',
+      clientIdTooltip:
+        'Vul hier het clientId in, dat geconfigureerd staat onder OpenZaak beheer onder API authorisaties > Applicaties, dat de juiste authorisaties heeft voor de benodigde functionaliteit',
       clientSecret: 'Secret',
+      clientSecretTooltip: 'Vul de secret in die hoort bij de clientId hierboven',
     },
     en: {
       title: 'OpenZaak',
       description:
         'OpenZaak is a production-worthy API platform that implements the API standard for case-oriented working (the ZGW APIs).',
       configurationTitle: 'Configuration name',
+      configurationTitleTooltip:
+        'Under this name, the plugin will be recognizable in the rest of the application',
       clientId: 'Client ID',
+      clientIdTooltip:
+        'Enter the clientId here, which is configured under OpenZaak management under API authorizations > Applications, which has the correct authorizations for the required functionality',
```

Comment:
Enter the clientId here, which is configured under OpenZaak management (API authorizations > Applications). This client id must have the correct authorizations for the required functionality